docs-site/content/zh/db/schema-sync.md

```
@@ -0,0 +1,180 @@
+
+## 故障排除
+
+### 钩子不工作
+
+1. 检查钩子脚本是否存在:
+
+```bash
+ls -la .claude/hooks/moai/handle-db-sync.sh
+```
+
+2. 验证执行权限:
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)